packages/docs-markdown/changelog.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.2.97 (August 13th, 2021)
+
+- Small update the metadata tree view in metadata explorer.
```

Comment:
```suggestion
- Small update to the metadata tree view in metadata explorer.
```

Reply:
Thanks for the typo correction @gewarren.